### DISCUSSION TITLE
get inline result

### DISCUSSION THREAD

**Discussion by 233-puchi** [microsoft/vscode-discussions#483] [Extension Development QnA] [answered]
During an extension test, how can I obtain the completed content by executing  “await vscode.commands.executeCommand ('editor.action.inlineSuggest.trigger');” to trigger inline completion?
Or which of the following built-in commands is similar to "(await vscode.commands.executeCommand(' vscode.executeInlineCompletionItemProvider', docUri, position)) as vscode.InlineCompletionItem; " to trigger and obtain inline completion.

**Comment by isidorn** [comment, marked as answer]
You can not get that as an extension. The API is not designed like that.
